test/Microsoft.AspNet.Localization.FunctionalTests/TestRunner.cs

```diff
@@ -0,0 +1,68 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved. 
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information. 
+
+using System;
+using System.IO;
+using System.Net;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Microsoft.AspNet.Server.Testing;
+using Microsoft.Extensions.Logging;
+using Xunit;
+
+namespace Microsoft.AspNet.Localization.FunctionalTests
+{
+    public class TestRunner
+    {
+        public async Task RunTestAndVerifyResponse(
```

Comment:
@kirthik seems ```TestRunner``` had been used in ```LocalizationTest```, so it will be nice if it becomes as inner class, but if you think long run your solution is better to allow other tests to use it